configuration.nix

```diff
@@ -30,6 +30,15 @@
   # default nix-bitcoin nodes offer outgoing connectivity.
   # services.clightning.autolisten = true;
 
+  ### LND
+  # Disable clightning and uncomment the following line in order to enable lnd,
+  # a lightning implementation written in Go.
+  # services.lnd.enable = assert (!config.services.clightning.enable); true;
+  # WARNING: If you use lnd, you should manually backup your wallet mnemonic
+  # seed. In order to do so, you can run the following command after the
```

Comment:
If you have this warning in here, it would be nice to also add the warning about the `channel.backup` file that is updated with every new channel that is created. Or is this communicated to the user at another point?

Reply:
Thanks for your review! The sole reason why we added this is because due to our unlocker/walletcreate script users do not see the warning about backing up their seed that they normally see. But we could optionally add a sentence about channel.backup too.

Reply:
Right, it does make sense to add a comment about channel.backup